packages/cli/test/integration/generators/relation.belongs-to.integration.js

```diff
@@ -507,19 +509,19 @@ describe('lb4 relation', function() {
       });
 
       it('repository has updated constructor', async () => {
-        const singleWordClassConstractor = [
+        const singleWordClassConstructor = [
           /public readonly customer: BelongsToAccessor<Customer, typeof Order\.prototype\.id>;\n/,
           /constructor\(@inject\('datasources\.db'\) dataSource: DbDataSource, @repository\.getter\('CustomerRepository'\) protected customerRepositoryGetter: Getter<CustomerRepository>,\) \{\n/,
-          /super\(Order, dataSource\);\n {4}this\.customer = this\.createBelongsToAccessorFor\('customer', customerRepositoryGetter,\);\n {2}\}\n/,
+          /super\(Order, dataSource\);\n {4}this\.customer = this\.createBelongsToAccessorFor\('customer', customerRepositoryGetter,\);\n/,
```

Comment:
These regexp-based tests are verify difficult to understand and work with. Let's switch to snapshots instead. See https://github.com/strongloop/loopback-next/pull/3788.

How to proceed:
1. Open a new PR that will migrate these tests to snapshots. Don't make any changes in the production code, this way we will know that no regression is accidentally introduced.
2. Rebase this PR on top of the first one. Update the snapshots to capture the new version of generated code.